internal/ui/desk.go

```diff
@@ -257,10 +257,23 @@ func (l *desktop) startSettingsChangeListener(settings chan fynedesk.DeskSetting
 	}
 }
 
+func (l *desktop) startFyneSettingsChangeListener(settings chan fyne.Settings) {
+	go func() {
+		for {
+			<-settings
+			l.updateBackgrounds(l.Settings().Background())
+		}
```

Comment:
Not a that important, but I think this would be a bit tidier if you ranged over the channel.
Something like this perhaps (I think this should work):
```go
		for range settings {
			l.updateBackgrounds(l.Settings().Background())
		}
```